### PR TITLE
adding remove procedure

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3027

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/manage-result-sources.md [hunk 1]
```diff
@@ -115,7 +115,23 @@ The default result source is **Local SharePoint Results**, but you can choose to
 
     
 2. On the **Manage Result Sources** page, point to the result source that you want to set as the default, select the arrow that appears, and then select **Set as Default**.
+
+## Delete a result source
+
+A result source may no longer be needed. The following procedure provides the steps for removing a result source.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-result-sources.md:121]:
```suggestion
A result source might be no longer needed. The following procedure provides the steps for removing a result source.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion only.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**